metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -207,13 +207,18 @@ object CommandHTMLFormat {
         commandId: String,
         arguments: List[String]
     ): String = {
+      def escape(args: String): String = {
+        // The lib used to convert markdown to html in sublime doesn't properly
+        // recognize URL encoding so we have to use hexadecimal html encoding
+        args.replaceAll("\"", "&#x22;")
+      }
```

Comment:
>No need to. I added escaping for double quotes and it fixed the problem. What other ones you think we should escape ? I would prefer if we used a library for this rather than manually escaping things.

If what you did @ayoub-benali  works then it should be good enough for our needs. I found this [answer on SO](https://stackoverflow.com/a/40942504) about escaping, so maybe we could escape `& < > " '` ? Followig should work
```scala
val toEscape = Set('"', '<', '>', '&', '\'')
def escape(args: String): String =
  args.flatMap {
    case char if toEscape.contains(char) => s"&#x${char.toHexString}"
    case char => char.toString
  }
```